Fix amulets and cobra crossbow attributes

Sh3Ns opened the pull request:
# Description

Amulet bug fix in enchanted werewolf amulet, enchanted theurgic amulet, time result (onDeEquip) and skill bug.
& register attirbutes cobra crossbow.

## Behaviour
### **Actual**

This PR is to fix amulets enchanted werewolf amulet, enchanted theurgic amulet and cobra crossbow

### **Expected**

Equip and unequip enchanted werewolf amulet or enchanted theurgic amulet and the time will now decrease as expected, and cobra crossbow attributes corresponding.

## Type of change

Please delete options that are not relevant.

  - [x] Bug fix (non-breaking change which fixes an issue)
  - [x] New feature (non-breaking change which adds functionality)
  - [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
  - [ ] This change requires a documentation update

## How Has This Been Tested

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

  - [x] Test A
  - [x] Test B

**Test Configuration**:

  - Server Version: Main
  - Client: 12.85
  - Operating System:  Windows 10 / Ubuntu 20.4

## Checklist

  - [x] My code follows the style guidelines of this project
  - [x] I have performed a self-review of my own code
  - [x] I checked the PR checks reports
  - [ ] I have commented my code, particularly in hard-to-understand areas
  - [ ] I have made corresponding changes to the documentation
  - [x] My changes generate no new warnings
  - [x] I have added tests that prove my fix is effective or that my feature works

liontts commented:
Hi, I just tested with the last two commits you sent about prey, well sometimes (not always) the player kicks when logging in, when I can log in when opening the system the player kicks the client instantly.

Kolisium commented:
Tested this on lastest version os repo and didn't work

Commit:
Fix Amulet bug in enchanted werewolf amulet time and skill cobra crossbow and enchanted theurgic amulet

Amulet bug fix in enchanted werewolf amulet, enchanted theurgic amulet, time result (onDeEquip) and skill bug.
& register attirbutes cobra crossbow.

dudantas commented on data/items/items.xml:
```diff
@@ -47222,10 +47222,15 @@
 	<item fromid="30374" toid="30392" name="RESERVED SPRITE"/>
-	<item id="30393" article="a" name="cobra crossbow">
+		<item id="30393" article="a" name="cobra crossbow">
+        <attribute key="weaponType" value="distance"/>
```
```suggestion
	<item id="30393" article="a" name="cobra crossbow">
		<attribute key="weaponType" value="distance"/>
```

dudantas commented on data/items/items.xml:
```diff
@@ -47222,10 +47222,15 @@
+		<item id="30393" article="a" name="cobra crossbow">
+        <attribute key="weaponType" value="distance"/>
 		<attribute key="slotType" value="two-handed"/>
+        <attribute key="ammotype" value="bolt"/>
```
```suggestion
		<attribute key="ammotype" value="bolt"/>
```

dudantas reviewed (changes requested):
I added some suggestions to fix the indentation. It's just commit.

Commit:
update suggestions to fix the indentation>

Commit:
Merge branch 'opentibiabr:main' into fix-attributes-amulet-and-cobra-crossbow

Commit:
up

Commit:
i

Commit:
g

Commit:
prof

omeranha committed:
Update items.xml

Commit:
Update items.xml

Commit:
Merge branch 'fix-attributes-amulet-and-cobra-crossbow' of https://github.com/painer15999/otservbr-global into fix-attributes-amulet-and-cobra-crossbow
